packages/ra-data-simple-prisma/src/audit/auditHandler.ts

```diff
@@ -12,27 +18,35 @@ export const auditHandler = async (options: AuditOptions, request: Request) => {
     request.body.method === "deleteMany"
   ) {
     for (const id of request.body.params.ids) {
-      //todo
+      await createAuditEntry(options, request, id);
     }
-  } else {
-    createAuditEntry(options, request);
+  } else if ("id" in request.body.params) {
+    await createAuditEntry(options, request, request.body.params.id);
+  } else if (created) {
+    await createAuditEntry(options, request, created.id);
   }
 
   return true;
 };
 
-const createAuditEntry = async (options: AuditOptions, request: Request) => {
-  const table = prisma[options.tableName];
+const createAuditEntry = async (
+  options: AuditOptions,
+  request: Request,
+  id: any
+) => {
   let data = {
     [options.tableColumns.action]: request.body.method.split(/(?=[A-Z])/)[0], //createMany => create
     [options.tableColumns.resource]: request.body.resource,
+    [options.tableColumns.payload]: JSON.stringify({
```

Comment:
what if you force the users to use JSON field instead of a text field?

Reply:
because I was using sqlite